apps/accounts/fixtures/scopes.json

```diff
@@ -85,12 +83,132 @@
+            "slug": "patient/ExplanationOfBenefit.r",
+            "group": 5,
+            "description": "ExplanationOfBenefit FHIR Resource",
+            "protected_resources": "[\n \n [\n    \"GET\",\n   \"/v[12]/fhir/ExplanationOfBenefit[/?].*$\"\n  ]\n]",
+            "default": "False"
+        }
+    },
+    {
+        "model": "capabilities.protectedcapability",
+        "pk": 12,
+        "fields": {
+            "title": "Search my Medicare claim information.",
+            "slug": "patient/ExplanationOfBenefit.s",
+            "group": 5,
+            "description": "ExplanationOfBenefit FHIR Resource",
+            "protected_resources": "[\n  [\n    \"GET\",\n   \"/v[12]/fhir/ExplanationOfBenefit[/]?$\"\n  ]\n]",
+            "default": "False"
+        }
+    },
+    {
+        "model": "capabilities.protectedcapability",
+        "pk": 13,
+        "fields": {
+            "title": "Read and search my Medicare claim information.",
+            "slug": "patient/ExplanationOfBenefit.rs",
+            "group": 5,
+            "description": "ExplanationOfBenefit FHIR Resource",
+            "protected_resources": "[\n  [\n    \"GET\",\n   \"/v[12]/fhir/ExplanationOfBenefit[/]?$\"\n  ],\n  [\n    \"GET\",\n   \"/v[12]/fhir/ExplanationOfBenefit[/?].*$\"\n  ]\n]",
+            "default": "False"
+        }
+    },
+    {
+        "model": "capabilities.protectedcapability",
+        "pk": 14,
+        "fields": {
+            "title": "Read my Medicare and supplemental coverage information.",
+            "slug": "patient/Coverage.r",
+            "group": 5,
+            "description": "Coverage FHIR Resource",
+            "protected_resources": "[\n \n [\n    \"GET\",\n   \"/v[12]/fhir/Coverage[/?].*$\"\n  ]\n]",
+            "default": "False"
+        }
+    },
+    {
+        "model": "capabilities.protectedcapability",
+        "pk": 15,
+        "fields": {
+            "title": "Search my Medicare and supplemental coverage information.",
+            "slug": "patient/Coverage.s",
+            "group": 5,
+            "description": "Coverage FHIR Resource",
+            "protected_resources": "[\n  [\n    \"GET\",\n   \"/v[12]/fhir/Coverage[/]?$\"\n  ]\n]",
+            "default": "False"
+        }
+    },
+        {
+        "model": "capabilities.protectedcapability",
+        "pk": 16,
+        "fields": {
+            "title": "Read and search my Medicare and supplemental coverage information.",
+            "slug": "patient/Coverage.rs",
+            "group": 5,
+            "description": "Coverage FHIR Resource",
+            "protected_resources": "[\n  [\n    \"GET\",\n   \"/v[12]/fhir/Coverage[/]?$\"\n  ],\n  [\n    \"GET\",\n   \"/v[12]/fhir/Coverage[/?].*$\"\n  ]\n]",
+            "default": "False"
+        }
+    },
+    {
+        "model": "capabilities.protectedcapability",
+        "pk": 17,
+        "fields": {
+            "title": "Patient launch context.",
+            "slug": "launch",
```

Comment:
This is `launch` here, but `launch/patient` in `create_blue_button_scopes`. Based on other changes, I _think_ `launch/patient` is correct, but check me on that before fixing this.